pnpm-lock.yaml

```diff
@@ -2918,6 +2918,7 @@ importers:
       '@fluidframework/eslint-config-fluid': ^5.1.0
       '@fluidframework/fluid-static': workspace:~
       '@fluidframework/local-driver': workspace:~
+      '@fluidframework/map': workspace:~
```

Comment:
Note: pnpm-lockfile changes are due to needing to import SharedMap from 'fluidframework/map'.  (See change description.)